server.js

```diff
@@ -21,7 +22,7 @@ app.command(
 );
 
 app.action(
-  /^(APP|REJ)_.*/,
+  /^(APP|NOAT|REJ)_.*/,
```

Comment:
Please add a comment prior to this explaining what each of these keywords mean, just in case it's not clear to a future contributor.